webapp/src/main/java/com/box/l10n/mojito/service/tm/TMTextUnitVariantRepository.java

```diff
@@ -14,6 +14,9 @@
 
     List<TMTextUnitVariant> findAllByLocale_IdAndTmTextUnit_Tm_id(Long localeId, Long tmId);
 
+    // List<TMTextUnitVariant> findAllByLocale_IdAndTmTextUnit_IdOrderByCreatedDateDesc(Long localeId, Long tmId);
+    List<TMTextUnitVariant> findAllByLocale_IdAndTmTextUnit_Id(Long localeId, Long tmTextUnitId);
+
```

Comment:
Remove the commented line and make sure your query is sorted by the date in descending order:
```
List<TMTextUnitVariant> findByTmTextUnitIdAndLocaleIdOrderByCreatedDateDesc(Long tmTextUnitId, Long localeId);
```